backend/controllers/emp.controller.js

```diff
@@ -0,0 +1,82 @@
+//declare a library of controller
+const db = require('../models');
+
+const employees = {};
+//declare all in function
+//request will be handled by route module
+
+//CRUD operation for model
+employees.insert = (req,res) => {
+    
+    
+    const Emp = {
+        firstName: req.body.firstName,
+        lastName: req.body.lastName,
+        age: req.body.age
+
+    };
+    
+    db.Employee.create(Emp)
+    .then(data=>{
+        res.send(data);
+    })
+    .catch(err => {
+        res.status(500).send({
+            message: err.message || 'error adding data to database'});
+    });
+
+};
+
+//retrieve instance
+    //retrieve all instances>>>
+employees.retrieveAll = (req,res)=> {

```

Comment:
CRUD operation not needed, we are building register and login page only.

Reply:
but is needed on webportal side to display all available users right?